webapp/channels/src/components/youtube_video/youtube_video.test.tsx

```diff
@@ -51,9 +53,11 @@ describe('YoutubeVideo', () => {
     test('should match init snapshot', () => {
         const store = mockStore(initialState);
         const wrapper = mount(
-            <Provider store={store}>
-                <YoutubeVideo {...baseProps}/>
-            </Provider>,
+            <IntlProvider locale='en'>
```

Comment:
can you please replace this with the util wrapper mountWithIntl https://github.com/mattermost/mattermost/blob/47c132e860e16787e0d3ea66f1453a2b3b5e73d8/webapp/channels/src/tests/helpers/intl-test-helper.tsx#L93 if you search for it, will find plenty of examples for it.